### DISCUSSION TITLE
Error when running weight greater than 1 with SDXL.

### DISCUSSION THREAD

**Discussion by Wonderflex** [muerrilla/stable-diffusion-NPW#10] [Q&A]
Getting the following error message when attempting to use the slider at anything except for 1 while using SDXL:

*** Error executing callback cfg_denoiser_callback for E:\SD\SD1_6_0\extensions\stable-diffusion-NPW\scripts\npw.pyit/s]
    Traceback (most recent call last):
      File "E:\SD\SD1_6_0\modules\script_callbacks.py", line 216, in cfg_denoiser_callback
        c.callback(params)
      File "E:\SD\SD1_6_0\extensions\stable-diffusion-NPW\scripts\npw.py", line 87, in denoiser_callback
        if uncond.shape[1] > self.empty_uncond.shape[1]:
    AttributeError: 'dict' object has no attribute 'shape'

![image](https://github.com/muerrilla/stable-diffusion-NPW/assets/112730878/42dfc468-08f4-4b06-b49a-edae06ffbfcf)

![image](https://github.com/muerrilla/stable-diffusion-NPW/assets/112730878/b5abfd7d-28fb-4bfd-a3ef-8347a7590ad4)

**Comment by muerrilla** [comment]
Should be working fine with SDXL now, if you still care. 😁
